back/src/companies/mutations/__tests__/renewSecurityCode.test.ts

```diff
@@ -45,8 +62,45 @@ describe("renewSecurityCode", () => {
 
     randomNumberMock.mockReturnValueOnce("1234").mockReturnValueOnce("2345");
 
+    getCompanyActiveUsersMock.mockReturnValueOnce([]);
     await renewSecurityCode("85001946400013");
 
     expect(randomNumberMock).toHaveBeenCalledTimes(2);
   });
+  it("should send a notification email to all users and return updated company", async () => {
+    companyMock.mockResolvedValueOnce({
+      securityCode: "1234"
+    });
+    randomNumberMock.mockReturnValueOnce("4567");
+
+    updateCompanyMock.mockReturnValueOnce({
+      siret: "85001946400013",
+      name: "Code en stock",
+      securityCode: "4567"
+    });
+
+    const users = [
+      { email: "john.snow@trackdechets.fr", name: "John Snow" },
+      { email: "arya.starck@trackdechets.fr", name: "Arya Starck" }
```

Comment:
This is not the correct spelling: https://gameofthrones.fandom.com/fr/wiki/Arya_Stark